### PR TITLE
Lesson 10

### PR DESCRIPTION
**Pull request by LuciaRider**

### CODE REVIEW

File: TripHouse/main.js [hunk 1]
```diff
@@ -51,17 +51,60 @@ const data = [
...
+const homesImgArrowRight = (homesContent.getElementsByClassName('content_img')[3]);
+function createArrowRigt() {
+  return `<img class="arrow_right" src="./img/arrow-right.png" alt="arrow-right">`
+};
+homesImgArrowRight.insertAdjacentHTML('beforeend', createArrowRigt())
+
+const homesImgArrowLeft = (homesContent.getElementsByClassName('content_img')[0])
+function createArrowLeft() {
+  return `<img class="arrow_left" src="./img/arrow-left.png" alt="arrow-leftt">`
+};
```
Comment (romos21) [inline TripHouse/main.js:78]: Why you add it with js when arrows are static?

File: TripHouse/main.js [hunk 2]
```diff
@@ -51,17 +51,60 @@ const data = [
...
 function createCard1(data) {
   return `
   <div class="content_img col-xs-6 col-sm-6">
     <img src="${data.imageUrl}" alt="${data.name}">
     <p class="accent-blue">${data.name}</p>
     <p class="secondary">${data.city}, ${data.country}</p>
   </div>`
+};
+data.forEach((element, index) => {
+  if (index < 4) {
```
Comment (romos21) [inline TripHouse/main.js:64]: It's better to move values like 4 into variables - cause it looks like an option that can be changed in the future

File: TripHouse/main.js [hunk 3]
```diff
@@ -51,17 +51,60 @@ const data = [
...
+
+// prev.addEventListener('click', function() {
+// 	position	+= slides[0].offsetWidth;
+// 	if (position >= -slider.offsetWidth + slides[0].offsetWidth) next.style.display = 'block';
+// 	if (position === 0) {
+// 		prev.style.display = 'none';
+// 		position = 0;
+// 	}
+// 	slider.style.transform = `translateX(${position}px)`;
+// })
```
Comment (romos21) [inline TripHouse/main.js:110]: Why did you commented listeners? Should I wait for you will finish it ? Or should I marge it now?

File: TripHouse/main.js (hunk 1 above)
Comment (LuciaRider) [reply TripHouse/main.js:78]: As I wrote earlier, I mixed up branches and 10 and 11 tasks are finished in branch 11, comments for branch 10 are made in branch 11

**Pull request merged**